django/template/backends/jinja2.py

```diff
@@ -61,3 +65,29 @@ def render(self, context=None, request=None):
             context['csrf_input'] = csrf_input_lazy(request)
             context['csrf_token'] = csrf_token_lazy(request)
         return self.template.render(context)
+
+
+def get_exception_info(exception):
+    """
+    Formats exception information for display on the debug page.
+    """
+    context_lines = 10
```

Comment:
Okay. Do you think it would be clearer if this returned a class instance, instead of a dict?

```
return TemplateExceptionDetail(
    name=...
   message=...
)
```

Then there would be an obvious place to document it.